moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1184,40 +1184,40 @@ class MoveGroup::MoveGroupImpl
 }
 }
 
-moveit::planning_interface::MoveGroup::MoveGroup(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
+moveit::planning_interface::MoveGroupInterface::MoveGroupInterface(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
```

Comment:
If you compile an arbitrary cpp file, e.g. `gcc -c -o foo.o foo.cpp` with this file:
```
namespace foo {
int func()
{ return 0; }
}
```

and then run `objdump -Ct foo.o` you see that the object file provides the symbol `foo::func`.
(Usually you only get to see the symbol names on linking errors..)
I consider it best practice to be able to find the complete symbol name in the original cpp file.
This is violated if you use namespace directives.